Marlin/src/pins/stm32f1/pins_BTT_SKR_MINI_E3_common.h

```diff
@@ -235,6 +235,70 @@
       #define TFTGLCD_CS                    PA9
 
     #endif
+  
+  #elif EITHER(FYSETC_MINI_12864_2_1, BTT_MINI_12864_V1)
```

Comment:
You only need to check for `FYSETC_MINI_12864_2_1` and this can apply to the MKS Mini 12864 lcd as well: https://github.com/MarlinFirmware/Marlin/blob/22b99a2e382034f951568dda8d2c70af06ddc47f/Marlin/src/inc/Conditionals_LCD.h#L34-L37

Reply:
Ah, thank you! I think VSCode didn't recognise it with just `FYSETC_MINI_12864_2_1`, but that makes sense given that `BTT_MINI_12864_V1` is defined as an alias. Going to make the changes soon :>

Reply:
VSCode tends to get confused due to all the macro use in Marlin.